docs/getting-started/install-kubernetes.md

```diff
@@ -7,19 +7,23 @@ description: >
 
 <!-- toc -->
 - [Installing kubectl gadget](#installing-kubectl-gadget)
-  * [Using krew](#using-krew)
-  * [Install a specific release](#install-a-specific-release)
-  * [Compile from source](#compile-from-source)
+  - [Using krew](#using-krew)
+  - [Install a specific release](#install-a-specific-release)
+  - [Compile from source](#compile-from-source)
 - [Installing in the cluster](#installing-in-the-cluster)
-  * [Quick installation](#quick-installation)
-  * [Choosing the gadget image](#choosing-the-gadget-image)
-  * [Deploy to specific nodes](#deploy-to-specific-nodes)
-  * [Deploying into a custom namespace](#deploying-into-a-custom-namespace)
-  * [Hook Mode](#hook-mode)
-  * [Specific Information for Different Platforms](#specific-information-for-different-platforms)
-    + [Minikube](#minikube)
+  - [Quick installation](#quick-installation)
+  - [Choosing the gadget image](#choosing-the-gadget-image)
+  - [Deploy to specific nodes](#deploy-to-specific-nodes)
+  - [Deploying into a custom namespace](#deploying-into-a-custom-namespace)
+  - [Hook Mode](#hook-mode)
+  - [Deploying with an AppArmor profile](#deploying-with-an-apparmor-profile)
+  - [Specific Information for Different Platforms](#specific-information-for-different-platforms)
+    - [Minikube](#minikube)
 - [Uninstalling from the cluster](#uninstalling-from-the-cluster)
 - [Version skew policy](#version-skew-policy)
+- [Installing `ig`](#installing-ig)
+  - [Install a specific release](#install-a-specific-release-1)
+  - [Compile from source](#compile-from-source-1)
```

Comment:
My editor did that automatically. I guess its a good edit nontheless?

Reply:
I'd prefer to avoid this as part of this PR. Also, which editor is it and why it's changing it? :scream:

Reply:
Its VSCode. Idk, It thinks it is prettier :D

Reply:
I remember some time ago I tried an extension/tool to automatically create (and update) those table of contents, but it never worked fine for me. It seems you have configured visual code to do it! 

I think the change is good, but better on a different PR / commit.

Reply:
https://github.com/inspektor-gadget/inspektor-gadget/pull/2749